### ISSUE TITLE
Retrieve SubjectSet by display_name

### ISSUE THREAD

**Issue by mkosmala** [zooniverse/Panoptes#707]
I'd like to do this:
GET https://panoptes-staging.zooniverse.org/api/subject_sets?project_id=62&display_name=cats

**Comment by edpaget** [comment]
With regards to this issue and #708, the subject set display name is intended to only be an aid to the project author, which is why it's not filterable and why there is no requirement for uniqueness. 

What's the use case for filtering on the server? Do you have many subject sets? You could always just filter the collection in your client since there probably aren't that may you're getting back

**Comment by mkosmala** [comment]
I have a handful of subject sets. Since I only "know" the display_name for the subject set and not the ID, it would simply be convenient to filter by display_name. For example, I set up my project and then later I decide I want to add subjects. Unless I've made a note of the ID, I have to pull all the subject sets and filter them myself. (Which is what I'm presently doing, BTW.) 

Let's see... I think I'll end up with 23 subject sets for this project. So it would be nice to be able to filter by display_name or else up the page size >20. (Oh, actually I'll have 25, since it appears that there are two default subject sets...)

WRT #708, I've made two subject sets with the same name, but now since I'm filtering in my scripts, I have no idea which subject set I 'want'. This is, I think, about not letting users shoot themselves in the foot.
